cloudevents/extensions/attrschema-schema.yaml

```diff
@@ -0,0 +1,97 @@
+title: "Attrschema meta-schema"
+$schema: "http://json-schema.org/draft-07/schema#"
+$id: "https://cloudevents.io/attrschema/2022-10/schema#"
+allOf:
+  - { "$ref": "http://json-schema.org/draft-07/schema#" }
+properties: # this is the properties of the JSON schema
+  type:
+    const: object
+  properties: # this is the properties property of the JSON schema
+    type: object
+    properties:
+      id:
+        type: object
+        properties:
+          cetype:
+            const: string
+      source:
+        type: object
+        properties:
+          cetype:
+            const: uri-reference
+      specversion:
+        type: object
+        properties:
+          cetype:
+            const: string
+      type:
+        type: object
+        properties:
+          cetype:
+            const: string
+      datacontenttype:
+        type: object
+        properties:
+          cetype:
+            const: string
+      dataschema:
+        type: object
+        properties:
+          cetype:
+            const: uri
+      subject:
+        type: object
+        properties:
+          cetype:
+            const: string
+      time:
+        type: object
+        properties:
+          cetype:
+            const: timestamp
+
+    patternProperties:
+      "^[a-z0-9]+$": # CloudEvents attribute names MUST consist of lower-case letters ('a' to 'z') or digits ('0' to '9') from the ASCII character set
+        type: object
+        allOf:
+          - type: object
+            properties:
+              type:
+                const: string # All context attribute values are normalized to string
+          - type: object
+            oneOf:
+              - properties:
+                  cetype:
+                    type: string
+                    enum:
+                      - boolean
+                      - integer
+                      - string
+                      - binary
```

Comment:
Is this saying that additional properties (are these for extensions?) can be either boolean, integer, string, or binary types?

Reply:
Yes,
Or URIs, references and timestamps